app/uk/gov/hmrc/customs/declaration/controllers/actionbuilders/batchFileUploadPayloadValidationActions.scala

```diff
@@ -78,7 +78,8 @@ class BatchFileUploadPayloadValidationComposedAction @Inject()(val batchFileUplo
             val files: Seq[BatchFileUploadFile] = (xml \ filesLabel \ "_").theSeq.collect {
```

Comment:
Also completely unrelated to your story could you do me a favour and make a small fix in my code:  in `CustomsDeclarationsMetricsService` can you rename `verifyFileTransmissionServiceWasCalledWith` to `verifyCustomsDeclarationsMetricsServiceWasCalledWith`, also delete `verifyCustomsDeclarationsMetricsServiceWasNotCalled` as it is not called

Reply:
Done